proposals/gc/MVP.md

```diff
@@ -748,42 +748,45 @@ The opcode for heap types is encoded as an `s33`.
 | Opcode | Type            | Parameters | Note |
 | ------ | --------------- | ---------- | ---- |
 | i >= 0 | `(type i)`      |            | from funcref proposal |
+| -0x0d  | `nofunc`        |            | |
+| -0x0e  | `noextern`      |            | |
+| -0x0f  | `none`          |            | |
 | -0x10  | `func`          |            | from funcref proposal |
 | -0x11  | `extern`        |            | from funcref proposal |
 | -0x12  | `any`           |            | |
 | -0x13  | `eq`            |            | |
-| -0x16  | `i31`           |            | |
-| -0x17  | `nofunc`        |            | |
-| -0x18  | `noextern`      |            | |
-| -0x19  | `struct`        |            | |
-| -0x1a  | `array`         |            | |
-| -0x1b  | `none`          |            | |
+| -0x14  | `i31`           |            | |
+| -0x15  | `struct`        |            | |
+| -0x16  | `array`         |            | |
 
 #### Structured Types
 
-| Opcode | Type            | Parameters |
-| ------ | --------------- | ---------- |
-| -0x21  | `struct ft*`    | `ft* : vec(fieldtype)` |
-| -0x22  | `array ft`      | `ft : fieldtype`       |
+| Opcode | Type            | Parameters | Note |
+| ------ | --------------- | ---------- | ---- |
+| -0x20  | `func t1* t2*`  | `t1* : vec(valtype)`, `t2* : vec(valtype)` | from Wasm 1.0 |
+| -0x21  | `struct ft*`    | `ft* : vec(fieldtype)` | |
+| -0x22  | `array ft`      | `ft : fieldtype`       | |
 
 #### Subtypes
 
 | Opcode | Type            | Parameters | Note |
 | ------ | --------------- | ---------- | ---- |
+| -0x20  | `func t1* t2*`  | `t1* : vec(valtype)`, `t2* : vec(valtype)` | shorthand |
 | -0x21  | `struct ft*`    | `ft* : vec(fieldtype)` | shorthand |
 | -0x22  | `array ft`      | `ft : fieldtype`       | shorthand |
 | -0x30  | `sub $t* st`    | `$t* : vec(typeidx)`, `st : strtype` | |
-| -0x32  | `sub final $t* st` | `$t* : vec(typeidx)`, `st : strtype` | |
+| -0x31  | `sub final $t* st` | `$t* : vec(typeidx)`, `st : strtype` | |
 
 #### Defined Types
 
 | Opcode | Type            | Parameters | Note |
 | ------ | --------------- | ---------- | ---- |
+| -0x20  | `func t1* t2*`  | `t1* : vec(valtype)`, `t2* : vec(valtype)` | shorthand |
 | -0x21  | `struct ft*`    | `ft* : vec(fieldtype)` | shorthand |
 | -0x22  | `array ft`      | `ft : fieldtype`       | shorthand |
 | -0x30  | `sub $t* st`    | `$t* : vec(typeidx)`, `st : strtype` | shorthand |
-| -0x31  | `rec dt*`       | `dt* : vec(subtype)` | |
-| -0x32  | `sub final $t* st` | `$t* : vec(typeidx)`, `st : strtype` | shorthand |
+| -0x31  | `sub final $t* st` | `$t* : vec(typeidx)`, `st : strtype` | shorthand |
+| -0x34  | `rec dt*`       | `dt* : vec(subtype)` | |
```

Comment:
It looks like the interpreter wasn't changed accordingly

Reply:
Oops, thanks. Fixed. Also changed the rec opcode to -0x32, not sure why I picked something else before.